strategy.cpp

```diff
@@ -0,0 +1,252 @@
+#include "strategy.h"
+#include "Pixy.h"
+
+//Constructor for the trategy class.
+Strategy::Strategy()
+{
+    robot_ = Movements();
```

Comment:
I don't see the point of redefining a Movements instance instead of using the existing one, the one that is declared in setup.